Make some minor language and formatting changes to the PR template (#6)
cisagov/.github: .github/pull_request_template.md

* [ ] _All_ future TODOs are captured in issues, which are referenced
      in code comments.
* [ ] All relevant type-of-change labels have been added.
* [ ] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [ ] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [ ] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.

> * [ ] Tests have been added and/or modified to cover the changes in this PR.

Reviewer: Would this be a simpler catch-all to use?
```suggestion
* [ ] Tests have been updated to cover the changes in this PR.
```

Reviewer 2: I think it's worth calling out "added" to put it in peoples' heads in case that hadn't occurred to them.

Author: I like the slightly more explicit language in the original.

Author: ![image](https://user-images.githubusercontent.com/5521725/98581472-87444c80-228f-11eb-969c-602ef039660d.png)
